packages/cucumber/package.json

```diff
@@ -53,7 +53,10 @@
   },
   "peerDependencies": {
     "@cucumber/cucumber": "^7.3.0",
-    "cucumber": "^1.3.3 || ^2.3.1 || ^3.2.1 || ^4.2.1 || ^5.0.0 || ^6.0.0"
+    "cucumber": "^1.3.3 || ^2.3.1 || ^3.2.1 || ^4.2.1 || ^5.0.0 || ^6.0.0",
```

Comment:
`peerDependencies` don't need changing I think; Those are modules require by a user of Serenity/JS to have.